### DISCUSSION TITLE
Add a line at the very top (right under the YYYY-MM-DD title)

### DISCUSSION THREAD

**Discussion by wsw70** [coddingtonbear/obsidian-local-rest-api#114] [Q&A] [answered]
I would like to prefix my daily note with a line (i.e. insert it below the title)

I thought that using `PATCH /periodic/{period}` could be a lotion but if I git it right, the position is always relative to a header.

Is there a way to use a position relative to the top of the file?

**Comment by coddingtonbear** [comment, marked as answer]
That would right now, unfortunately, require that you assemble the content manually with a `GET` followed by a `PUT`. That’s one of the many shortcomings of the current `PATCH` method.  Good news, though— there are improvements in the works for that: https://github.com/coddingtonbear/obsidian-local-rest-api/wiki/Flexible-PATCH.

**Reply by wsw70** [reply]
Thank you. The idea of

> you assemble the content manually with a GET followed by a PUT.

is a good one, and easy to do so I will go for that, thanks!
